tests/integration_tests/test_examples.py

```diff
@@ -151,3 +151,48 @@ def test_pearl_ml1_push():
         '--embedding_mini_batch_size', '2', '--max_path_length', '1'
     ],
                           check=False).returncode == 0
+
+
+@pytest.mark.nightly
+@pytest.mark.no_cover
+@pytest.mark.timeout(120)
+def test_maml_ml10():
```

Comment:
please have @gitanshu review this

adding new special test cases is generally discouraged. most example tests should be self-contained and run by virtue of living in examples/

Reply:
I need to also set `meta_batch_size` to 1 to reduce time. Do you have an idea to achieve this without making a special test case?

Reply:
How long does it take to run 1 epoch without any other parameter change?

Reply:
105s on my local machine.